src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -61,22 +61,38 @@ private Storage() {
 
     mTopRoller.setNeutralMode(NeutralMode.Brake);
     mBottomRoller.setNeutralMode(NeutralMode.Brake);
-    
-    mBottomRoller.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 10);
-    mBottomRoller.setSensorPhase(false);
 
     mIntakeSensor = new DigitalInput(INTAKE_SENSOR_PORT);
+    
+    if (Robot.getIsPracticeBot()) {
+      BALL_DISTANCE_IN_ENCODER_TICKS = Config.getInstance().getDouble(Key.STORAGE__BALL_DISTANCE_IN_ENCODER_TICKS_PRACTICE);
+      mBottomRoller.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 10);
+      mBottomRoller.setSensorPhase(false);
+    } else {
+      BALL_DISTANCE_IN_ENCODER_TICKS = Config.getInstance().getDouble(Key.STORAGE__BALL_DISTANCE_IN_ENCODER_TICKS_PRODUCTION);
+      mTopRoller.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 10);
+      mBottomRoller.setSensorPhase(false);
```

Comment:
Why do you set the phase on the bottom roller here?

Reply:
Not gonna lie, thought I explicitly fixed that.
- liam